apps/web/src/ee/clerk/EnterpriseAuthRoutes.tsx

```diff
@@ -0,0 +1,41 @@
+import { SignedIn, SignedOut } from '@clerk/clerk-react';
+import { Navigate, Route } from 'react-router-dom';
+import { PublicPageLayout } from '../../components/layout/components/PublicPageLayout';
+import { ROUTES } from '../../constants/routes';
+import OrganizationListPage from './pages/OrganizationListPage';
+import QuestionnairePage from './pages/QuestionnairePage';
+import SignInPage from './pages/SignInPage';
+import SignUpPage from './pages/SignUpPage';
+
+export const EnterpriseAuthRoutes = () => {
+  const EnterprisePublicAuthLayout = () => (
+    <SignedOut>
+      <PublicPageLayout />
+    </SignedOut>
+  );
+
+  // private but we want appearance of public layout
+  const EnterprisePrivateAuthLayout = () => (
+    <>
+      <SignedIn>
+        <PublicPageLayout />
+      </SignedIn>
+      <SignedOut>
+        <Navigate to={ROUTES.AUTH_LOGIN} replace />
+      </SignedOut>
+    </>
+  );
+
+  return (
+    <>
+      <Route element={<EnterprisePublicAuthLayout />}>
+        <Route path={ROUTES.AUTH_SIGNUP + '/*'} element={<SignUpPage />} />
```

Comment:
Let's use `/sign-in` and `/sign-up` in the ee edition.

Reply:
The `/auth/login` URL is used and often times hardcoded on multiple places outside of web - I would rather not do this change as part of this PR since it would increase the scope and it would be harder to track the issues related to this path change.